client/src/components/HeroAbout.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// import bgimage from "";
+// style={{ backgroundImage: `url(${bgimage})`, backgroundSize: 'cover' }}
+
+const Hero = (props) => {
+  return (
+    <div className="heroSection3">
```

Comment:
these two classes should match the component name

Reply:
Done